gui/src/components/HomeComponents/LandingPage.tsx

```diff
@@ -31,6 +71,113 @@ export default function LandingPage() {
     }
   }
 
+  const onSetEmail = (value: string) => {
+    setEmail(value);
+    setIsEmailRegistered(null);
+    setPassword('');
+    setShowPassword(false);
+    setEmailErrorMsg('');
+  };
+
+  const onSetPassword = (value: string) => {
+    setPassword(value);
+    setPasswordErrorMsg('');
+  };
+
+  const toSetUserData = (user, access_token: string) => {
+    const userData: userData = {
+      userEmail: user.Email,
+      userName: user.Name,
+      organisationId: user.OrganisationID,
+      accessToken: access_token,
+    };
+    setUserData(userData);
+  };
+
+  async function toCheckUserEmail() {
+    try {
+      if (!validateEmail(email)) {
+        setEmailErrorMsg('Enter a Valid Email.');
+        return;
+      }
+      setIsButtonLoading(true);
+      const response = await checkUserEmailExists(email);
+      if (response) {
+        const data = response.data;
+        if (data.user_exists) {
+          setIsEmailRegistered(true);
+        } else {
+          setIsEmailRegistered(false);
+        }
+      }
+      setIsButtonLoading(false);
```

Comment:
add this in the finally block instead

Reply:
@Parth2506 added to finally block.
Resolved.